stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -126,7 +126,7 @@ pub enum Error {
     MinerAborted,
     ChannelClosed(String),
     /// This error indicates a Epoch2 block attempted to build off of a Nakamoto block.
-    InvalidChildOfNakomotoBlock,
+    InvalidChildOfNakamotoBlock,
```

Comment:
I'm not sure if it's safe to change this or if it will be consensus-breaking. I'll need to come back to this.

Reply:
Hi @obycode ,did you check? I was waiting for your reply.